### DISCUSSION TITLE
Is it possible to export exploded/scanned files

### DISCUSSION THREAD

**Discussion by nadrojisk** [target/strelka#457] [Q&A] [answered]
Does strelka save the exploded and uploaded files in a way that can later be exported by users interacting with it?

**Comment by phutelmyer** [comment, marked as answer]
@nadrojisk Yes, in a way.

If you uncomment [`ScanSave`](https://github.com/target/strelka/blob/master/configs/python/backend/backend.yaml#L538-L545) in the `backend.yml` configuration prior to starting Strelka, the [`ScanSave`](https://github.com/keiche/strelka/blob/master/src/python/strelka/scanners/scan_save.py) scanner created by @keiche will output the contents of each matching file to a compressed and encoded string.

Original Pull Request: https://github.com/target/strelka/pull/362

By taking the output from `scansave.file`, you can convert that from the encoded string to the original file.

If you would like more assistance, let me know and I can provide a better example.
